ci_framework/roles/devscripts/tasks/sub_tasks/_521_gather.yml

```diff
@@ -39,7 +40,7 @@
             {{
               ansible_facts[item]['ipv4']['network'] +
               '/' +
-              ansible_facts[item]['ipv4']['network']
+              ansible_facts[item]['ipv4']['prefix']
```

Comment:
iirc we can use the ipaddr filter: https://docs.ansible.com/ansible/latest/collections/ansible/utils/docsite/filters_ipaddr.html#converting-subnet-masks-to-cidr-notation

Might be a bit stronger?

Reply:
We could use `ipaddr` however the code would modify like

```
{{
  ( ansible_facts[item]['ipv4']['network'] +
  '/' +
  ansible_facts[item]['ipv4']['netmask'] ) |
  ansible.builtin.ipaddr('network/prefix')
}}
```

since the prefix is already available, I believe we could avoid `ipaddr` here